dip-0003.md

```diff
@@ -246,7 +244,7 @@ Updating is done on the so called chain-tips list, which is the list valid at th
 
 The chain-tips list is updated for every new block that is processed. The rules to update the list are the following:
 
-  1. If a valid ProRegTx is encountered, a masternode entry (identified by the hash of the ProRegTx) is added to the registered set. The entry also contains metadata (IP address, public keys, etc.) found in the ProTx.
+  1. If a valid ProRegTx is encountered, a masternode entry (identified by the hash of the ProRegTx) is added to the registered set. The entry also contains metadata (masternode network information, public keys, etc.) found in the ProTx.
```

Comment:
nit: `masternode network information` sounds a bit too generic; I think it's better to use words such as `masternode address information` or masternode network address`?
It's address and information how to connect; not just some abstract network information IMO

Reply:
Renamed it to "masternode addresses", resolved in latest push!